README.md

```diff
@@ -892,6 +892,33 @@ using (var transaction = database.BeginTransaction(IsolationLevel.Chaos))
 }
 ```
 
+### Post-migration actions
+Option for post-migration actions and post-migration database views refresh with an option to accept the script ID up to which actions will be executed after migration is available. The default script RefreshViews.sql for refreshing views is in resources.
```

Comment:
Daj prázdnu medzeru pod nadpis.
Ideálne keď markdown edituješ vo VSCode a nainštaluješ si Markdown linter, alebo Markdown all in one.

Markdown má totižto svoje pravidlá, ktoré je potrebné dodržiavať aby sa správne napríklad generoval HTML obsah, ...
A tento linter ti to bude strážiť a upozorňovať ťa na to.